aviatrix_controller_initialize/aviatrix_controller_init.py

```diff
@@ -26,16 +32,27 @@ def function_handler(event):
     account_email = event["account_email"]
     access_account_name = event["access_account_name"]
     aviatrix_customer_id = event["aviatrix_customer_id"]
-    controller_init_version =event["controller_init_version"]
+    controller_init_version = event["controller_init_version"]
     wait_time = default_wait_time_for_apache_wakeup
 
     # Reconstruct some parameters
     aviatrix_customer_id = aviatrix_customer_id.rstrip()
     aviatrix_customer_id = aviatrix_customer_id.lstrip()
-    api_endpoint_url = "https://" + ucc_public_ip + "/" + aviatrix_api_version + "/" + aviatrix_api_route
+    api_endpoint_url = (
+        "https://"
+        + ucc_public_ip
+        + "/"
+        + aviatrix_api_version
+        + "/"
+        + aviatrix_api_route
+    )
 
     # Step1. Wait until the rest API service of Aviatrix Controller is up and running
-    print(keyword_for_log + "START: Wait until API server of Aviatrix Controller is up and running")
+    print(
+        keyword_for_log
```

Comment:
Would be better to initialize a logger and set the format,  rather than pass the keyword_for_log everywhere